pkg/ddc/alluxio/worker.go

```diff
@@ -196,3 +143,112 @@ func (e *AlluxioEngine) getWorkerSelectors() string {
 	}
 	return selectorValue
 }
+
+// buildWorkersAffinity builds workers affinity if it doesn't have
+func (e *AlluxioEngine) buildWorkersAffinity(workers *v1.StatefulSet) (workersToUpdate *v1.StatefulSet, err error) {
```

Comment:
I think we don't need this function anymore.

Reply:
thanks, buildWorkersAffinity of Alluixo worker.go  deleted